main.py

```diff
@@ -68,45 +67,39 @@ async def query_virtual_agent(inputs: QueryVLAInputs):
     # make sure the VLA is enabled
     enable_vla(inputs.client_id, inputs.group_id)
 
-    redis_key = f"{inputs.community_id}:{inputs.client_id}"
-    redis_state = redis.get(redis_key)
-
-    conversation_history = json.loads(redis_state) if redis_state else []
-
-    # most deterministic results
-    temperature = 0
-
-    # create memory
-    messages = []
-    for interaction in conversation_history:
-        messages.extend([
-            HumanMessage(content=interaction['human_message']),
-            AIMessage(content=interaction['ai_message'])
-        ])
+    # get conversation history
+    messages = State(inputs.community_id, inputs.client_id).get_memory_of_messages()
```

Comment:
Is this used anywhere?

Reply:
Yes, it's passed into the agent down below.